assignment-5/src/store.js

```diff
@@ -0,0 +1,67 @@
+import Vue from "vue";
```

Comment:
this should be named as index.js and kept in store folder like we doing for router